src/daft-local-execution/src/lib.rs

```diff
@@ -14,6 +16,13 @@ lazy_static! {
     pub static ref NUM_CPUS: usize = std::thread::available_parallelism().unwrap().get();
 }
 
+fn get_morsel_size() -> usize {
+    env::var("DAFT_MORSEL_SIZE")
+        .unwrap_or_else(|_| "1000".to_string())
```

Comment:
its perferable to have the default size be a static const value

`const DEFAULT_MORSEL_SIZE: usize = 1000`